backend/ethereum_auth/tests.py

```
@@ -249,9 +249,9 @@ def _authenticate(self, user):
         refresh = RefreshToken.for_user(user)
         self.client.credentials(HTTP_AUTHORIZATION=f'Bearer {refresh.access_token}')
 
-    def _start_pending_email(self, pending, *, email='new@example.com', token='signup-token', name='New User'):
+    def _start_pending_email(self, pending, *, email='new@example.com', code='123456', name='New User'):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_start_pending_email`

(ANN202)